hadoop-yarn-project/hadoop-yarn/hadoop-yarn-server/hadoop-yarn-server-nodemanager/src/main/java/org/apache/hadoop/yarn/server/nodemanager/containermanager/ContainerManagerImpl.java

```diff
@@ -212,6 +213,10 @@
   private final Map<ContainerId, Future> x509Updaters = new HashMap<>();
   private final Map<ContainerId, Future> jwtUpdaters = new HashMap<>();
 
+  private String keyStoreFileName = SSLFactory.DEFAULT_LOCALIZED_KEYSTORE_FILE_NAME;
+  private String trustStoreFileName = SSLFactory.DEFAULT_LOCALIZED_TRUSTSTORE_FILE_NAME;
+  private String passwordFileName = SSLFactory.DEFAULT_LOCALIZED_PASSWD_FILE_NAME;
```

Comment:
These can be final

Reply:
No they can't because they are initialized in the service init with the configured value. I don't really need to put the default value here.

Reply:
OK